packages/blitz-paint/src/text.rs

```diff
@@ -87,3 +87,31 @@ pub(crate) fn stroke_text<'a>(
         }
     }
 }
+
+/// Draw selection highlight rectangles for the given byte range in a layout.
+/// Uses Parley's Selection type for accurate geometry calculation.
+pub(crate) fn draw_text_selection(
+    scene: &mut impl PaintScene,
+    layout: &Layout<TextBrush>,
+    transform: Affine,
+    selection_start: usize,
+    selection_end: usize,
+) {
+    let anchor = Cursor::from_byte_index(layout, selection_start, Affinity::Downstream);
+    let focus = Cursor::from_byte_index(layout, selection_end, Affinity::Downstream);
+    let selection = Selection::new(anchor, focus);
+
+    let selection_color = color::palette::css::STEEL_BLUE;
```

Comment:
Can we change this color to `Color::from_rgb8(180, 213, 255)`? This matches the selection color in Safari and Firefox use (and is very close to the color Chrome uses) on macOS (can we also switch this color for text input selections).

I realise that the `STEEL_BLUE` color was already in use, but it looks bad when selecting page text.